ui/src/app/components/object-explorer/object-explorer.component.scss

```diff
@@ -3,6 +3,34 @@
   .filter-wrapper {
     padding: 0 10px;
   }
+  .mat-header-cell-name {
```

Comment:
These css components seem to be generic see if they can be moved out to generic css file instead of component specific one

Reply:
Generic css are moved to global scss file